content/zh-cn/docs/reference/kubernetes-api/workload-resources/resource-claim-template-v1alpha3.md

```diff
@@ -67,20 +71,22 @@ ResourceClaimTemplateSpec 包含针对 ResourceClaim 的元数据和字段。
 <hr>
 
 <!--
-- **spec** (<a href="{{< ref "#ResourceClaimTemplateSpec" >}}">ResourceClaimSpec</a>), required
+- **spec** (<a href="{{< ref "../workload-resources/resource-claim-v1alpha3#ResourceClaimSpec" >}}">ResourceClaimSpec</a>), required
```

Comment:
This dependency is relatively big and need some time to add.

Reply:
> This dependency is relatively big and need some time to add.

Shall we hold this till the depended one gets merged?

Reply:
/hold
ok, wait for #47783 get merged.

Reply:
/unhold